### ISSUE TITLE
gawk still uses /bin/sh

### ISSUE THREAD

**Issue by sp4rkie** [termux/termux-packages#562]
I'm currenly playing around with gawk (my favorite scripting tool). I noticed that
`awk 'BEGIN { while ("date" | getline line > 0) print line; }'`
unfortunately does not work. This is because:
```
1088 u0_a88@localhost[files/home] > strace -f awk 'BEGIN { while ("date" | getline line > 0) print line; }' 2>&1|egrep execve
execve("/data/data/com.termux/files/usr/bin/awk", ["awk", "BEGIN { while (\"date\" | getline "...], [/* 25 vars */]) = 0
[pid 14698] execve("/bin/sh", ["sh", "-c", "date"], [/* 25 vars */] <unfinished ...>
[pid 14698] <... execve resumed> )      = -1 ENOENT (No such file or directory)
```
still uses /bin/sh. I'll try to setup a build environment on my current debian in the hope I can fix that and issue a commit. Caution: I haven't dealt yet with github or android development :-)

**Comment by fornwall** [comment]
Thanks! This has been fixed in version `4.1.4-1` of the gawk package, update with:

    apt update && apt upgrade

**Comment by sp4rkie** [comment]
Great! Works as expected now.
